### ISSUE TITLE
fix arg parsing

### ISSUE THREAD

**Issue by ConorSheehan1** [ConorSheehan1/homebrew-ConorSheehan1#2]
fire CLIs installed as package not script. e.g.
```python
# -*- coding: utf-8 -*-
import re
import sys
from shot import shot
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(shot())
```

this means the default action is always run.

## possible solutions
either:
1. change script to run `python shot.py`
2. switch to argparse etc so `shot()` parses args
3. build binary and [distribute as bottle](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/)

**Comment by ConorSheehan1** [comment]
Hacky fix locally
```bash
#!/bin/bash
# use python installed by formula, to run script installed by formula, pass all args from bash to python with `$@`
/home/linuxbrew/.linuxbrew/Cellar/shot/0.1.0/libexec/bin/python /home/linuxbrew/.linuxbrew/Cellar/shot/0.1.0/libexec/lib/python3.9/site-packages/shot.py $@
```

**Comment by ConorSheehan1** [comment]
Closed by 7c27754ceb32216955f373e116789c9f322b9110
First fix using method 1. `change script to run python shot.py` 
https://engineering.innovid.com/distributing-command-line-tools-with-homebrew-d03e795cadc8

Possibly second fix in next version of shot: add main function same as osxdocker.
